eras/alonzo/impl/src/Cardano/Ledger/Alonzo/PlutusScriptApi.hs

```diff
@@ -198,7 +198,7 @@ collectTwoPhaseScriptInputs ei sysS pp tx utxo =
       case txinfo lang of
         Right inf -> Right (script, getData tx utxo sp ++ (d : [valContext inf sp]), eu, costs Map.! lang)
         Left te -> Left $ BadTranslation te
-    apply _ (_, _, eu) script = Right (script, [], eu, CostModel mempty)
+    apply _ (_, _, eu) script = Right (script, [], eu, CostModelV1 mempty undefined)
```

Comment:
It is no longer easy to make a `mempty` like cost model, since `mkEvaluationContext` can return Nothing. We'll need a plan for this...

Reply:
this branch is entirely unneeded with a bit of rearrangement.